### DISCUSSION TITLE
App control manager: Win10?

### DISCUSSION THREAD

**Discussion by bladehstream** [HotCakeX/Harden-Windows-Security#447] [General]
As an extension to https://github.com/HotCakeX/Harden-Windows-Security/issues/435#issue-2709440109

How much extra difficulty is required to allow operation in Win10 (22H2+) Enterprise/Pro?  Happy to test if that helps.

**Comment by HotCakeX** [comment]
The most important component is `CiTool.exe`, i don't remember it being available in Windows 10. Without it, the necessary logic would need to be implemented natively and that'd take some time, but given the fact that Windows 10 is set to reach end of life by the end of 2025 i don't know if it'll be worth the time, time that i can spend on improving the application with more features.

**Reply by bladehstream** [reply]
Citool.exe is definitely there and there are definitely a lot of organisations that will be buying ESUs 😄

Perhaps policy can be developed on win11 then copied across to 10 because functionally it's basically the same? I might give that a go

**Reply by HotCakeX** [reply]
For the scenarios where organizations manage devices with Windows 10 installed and use ESUs can use AppControl Manager on a Windows 11 device or VM, and remotely deploy the crafted policies via Intune.

I'll try it on Windows 10 see how it performs but implementing new features is more important for me. I'm currently internalizing all of the external function the app relies on, once that is completely done it can improve OS compatibility more.

At the moment the app can be used on Windows 11 22H2, 23H2, 24H2 and Windows Server 2025.
